.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "intelephense.environment.includePaths": ["/var/www/html/yetopen/basic/vendor"]
```

Comment:
do not commit IDE config files, remove it from git and add the file in .gitignore